boards/microbit_v2/README.md

```diff
@@ -118,6 +137,8 @@ To install apps by automatically detecting the Microbit v2:
 $ tockloader install app.tab
 ```
 
+Tockloader still uses openocd. Make sure you have the latest version installed.
```

Comment:
Given that we still have other openocd references, would it be better to keep the existing openocd directions, and add the new probe.rs material as a separate option?